wmfdata/presto.py

```diff
@@ -39,15 +39,26 @@ def run(commands, catalog="analytics_hive"):
         ca_bundle="/etc/ssl/certs/wmf-ca-certificates.crt"
     )
 
-    connection = prestodb.dbapi.connect(
-        catalog=catalog,
-        host="an-coord1001.eqiad.wmnet",
-        port=8281,
-        http_scheme="https",
-        user=USER_NAME,
-        auth=PRESTO_AUTH,
-        source=f"{USER_NAME}, wmfdata-python"
-    )
+    try:
```

Comment:
Can we add a comment here to explain the temp code?Can we add a comment here to explain the temp code?